city/crud.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.future import select
+from . import models, schemas
+
+
+async def create_city(db: AsyncSession, city: schemas.CityCreate):
```

Comment:
Ensure that `schemas.CityCreate` is correctly defined in the `schemas` module. It should match the expected input structure for creating a `City` instance.